### ISSUE TITLE
Incorrect serialization to JSON for entity with both regular and additional data

### ISSUE THREAD

**Issue by cdtj** [microsoftgraph/msgraph-sdk-go#174]
Comma missing when Entity has factory based fields and additional data at the same time:

```
requestBody := models.NewUserScopeTeamsAppInstallation()
// setting factory based attr
requestBody.SetTeamsApp(app)
requestBody.SetId(app.GetId())
teamsAppPtr := fmt.Sprintf("https://graph.microsoft.com/1.0/appCatalogs/teamsApps/%s", *app.GetId())
// setting additional data
requestBody.SetAdditionalData(map[string]interface{}{
	"teamsApp@odata.bind": &teamsAppPtr,
})
...
bodyBytes, _ := myWriter.GetSerializedContent() // ([]byte, error)
log.Println("bodyBytes:", string(bodyBytes))
```

beautified result:
```
{
"id": "##ID",
"teamsApp@odata.bind": "https://graph.microsoft.com/1.0/appCatalogs/teamsApps/##ID" <<< comma missing
"teamsApp": {
	"id": "##ID",
	"displayName": "##NAME",
	"distributionMethod": "organization",
	"externalId": "##EXT_ID"
}
}
```

fixing missing comma and performing curl request returns correct result

**Comment by baywet** [comment]
For other people running into this issue, the fix was just released and can be pulled using 

```shell
go get github.com/microsoft/kiota-serialization-json-go@v0.5.1
```

We'll update the dependency in the next releases.

**Comment by baywet** [comment]
released by #186

```shell
go get gitub.com/microsoftgraph/msgraph-sdk-go@v0.25.0
```

Should bring the update along.

Closing.
